.circleci/config.yml

```diff
@@ -1,23 +1,31 @@
-defaults: &defaults
+checkout_submodule: &checkout_submodule
+  run:
+    name: Checkout submodule
+    command: |
+        git submodule sync
+        git submodule update --init --recursive
+
+npm_cache_key: &npm_cache_key
+  npm-deps-v1-{{ checksum "package.json" }}
+
+restore_npm_cache: &restore_npm_cache
+  restore_cache:
+    keys:
+      - *npm_cache_key
+      - npm-deps-v1-
```

Comment:
Isn't this already include in `*npm_cache_key`?

Reply:
@kevinbarabash `npm_cache_key` includes the checksum of the file `package.json`, so if `package.json` changes, it will not match old caches.